tests/generators/compliance_runners/fork_choice/README.md

```diff
@@ -0,0 +1,53 @@
+# Fork choice compliance test generator
+
+Fork Choice test generator intended to produce tests to validate conformance to the specs of various Fork Choice implementations.
+
+Implementation of the approach described in the [Fork Choice compliance testing framework](https://hackmd.io/@ericsson49/fork-choice-implementation-vs-spec-testing).
+
+Preliminary research has been also performed in this [repo](https://github.com/txrx-research/fork_choice_test_generation/tree/main).
+
+To simplify adoption of the tests, we follow the test format described in the [fork choice test formats documentation](../../../formats/fork_choice/README.md), with a minor exception (new check added).
+
+This work was supported by a grant from the Ethereum Foundation.
+
+# Pre-requisites
+
+Install requirements (preferably, in a dedicated Python environment)
+
+```
+> pip install -r requirements.txt
+```
+
+In order to run tests, install `tqdm` additionally.
+```
+> pip install tqdm
+```
+
+# Generating tests
+
+```
+> python test_gen.py -o ${test_dir} --fc-gen-config ${config_dir}/test_gen.yaml
+```
```

Comment:
* I would prefer for the new requirement (`minizinc`) be added to `pyproject.toml` for consistency.
* It would be best if these could be executed with `make` (like `make reftests`) from the main Makefile.
* Why don't we include `tqdm` in pyproject too, instead of manually installing it?

Reply:
I've added `minizinc` and `tqdm` into `pyproject.toml`.

Working on implementing `make comptests`.